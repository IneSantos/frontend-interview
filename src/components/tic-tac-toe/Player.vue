<template>
    <div>
      <h1>{{name}}</h1>
      <h1>{{score}}</h1>
    </div>
</template>

<script>
  export default {
    name:'Player',
    props: {
        name: String,
        score: Number
    }
  }
</script>