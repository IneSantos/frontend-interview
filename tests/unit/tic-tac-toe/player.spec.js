import { shallowMount } from '@vue/test-utils'
import Player from '@/components/tic-tac-toe/Player.vue'

describe('Player.vue', () => {
  it('renders default', () => {
    const wrapper = shallowMount(Player);
    expect(wrapper).toMatchSnapshot();
  })
})
