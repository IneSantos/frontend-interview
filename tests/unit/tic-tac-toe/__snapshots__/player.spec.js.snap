// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player.vue renders default 1`] = `
<div class="player">
  <h1></h1>
  <p></p>
</div>
`;
